plugins/win-capture/window-helpers.c

```diff
@@ -383,6 +383,64 @@ static int window_rating(HWND window, enum window_priority priority,
 	return val;
 }
 
+static int window_rating_by_list(HWND window, const DARRAY(struct game_capture_picking_info) * games_whitelist)
+{
+	struct dstr cur_class = {0};
+	struct dstr cur_title = {0};
+	struct dstr cur_exe = {0};
+	int final_val = 0x7FFFFFFF;
```

Comment:
Wondering why we keep using this weird value here and other places? Why does it have to be this precise valued?

Reply:
it is "-1" and used as "no match"/"not set" for window rating. 
"0" is perfect match and anything more than "0" for windows with not perfectly match title.